packages/react-compose/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import hoistStatics from 'hoist-non-react-statics';
+import {NonReactStatics} from 'packages/useful-types/src';
```

Comment:
I know this is the automatic path VSCode adds, but it doesn't actually work once deployed. You'll need to change this to `@shopify/useful-types`

Reply:
Oh. I missed that. Fixing it now.